tests/mailer_test.py

```diff
@@ -1,17 +1,28 @@
+"""
+Tests that require a working mail server.
+Local mail server provided with a localmail server, which we create in the fixture.
+
+"""
+
+
 from jinja2.nativetypes import NativeEnvironment
 import time
-import mailer
 import sys
 import os
 import uuid
 import logging
 import pytest
 import configparser
+import threading
 
 from os.path import abspath, dirname, join
 
+from fixtures.localmail_config import localmail_config
+
 sys.path.append(dirname(dirname(abspath(__file__))))
 
+import mailer
```

Comment:
Did you move this import mailer after the sys.path.append for some reason? It's an unusual ordering, so if so, perhaps worth a comment as to why?

Reply:
mailer is `mailer.py` in the root. If we include it before the path is added, you can't run the test from within the `tests/` directory, only from the root directory.